Core/Core/Files/Model/LocalFileURLCreator.swift

```diff
@@ -35,7 +35,7 @@ extension LocalFileURLCreator {
         if mimeClass == "pdf" {
             // If the user already downloaded and modified the file locally, we don't want to download it again.
             // Instead, return the url pointing to the locally modified version.
-            let docsURL = URL.Directories.documents.appendingPathComponent(fileName)
+            let docsURL = URL.Directories.annotatedPDFs.appendingPathComponent(fileName)
```

Comment:
Modifying the load url like this will break access to already existing annotated files in the documents directory when the user updates the app to this version. Let's check for the annotated file on the original url as well OR create a job that moves all annotated pdfs to their new directory.